--- a/lib/rapi/rlib2.py 2022-05-28 18:26:53.172108336 -0700
+++ b/lib/rapi/rlib2.py 2022-05-28 18:28:34.326576340 -0700
@@ -91,6 +91,7 @@
             "beparams", "hvparams",
             "oper_state", "oper_ram", "oper_vcpus", "status",
             "custom_hvparams", "custom_beparams", "custom_nicparams",
+            "custom_osparams",
             ] + _COMMON_FIELDS

 N_FIELDS = ["name", "offline", "master_candidate", "drained",